firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/metadata/MetaDataStoreTest.java

```diff
@@ -14,15 +14,19 @@
 
 package com.google.firebase.crashlytics.internal.metadata;
 
+import com.google.common.truth.Truth;
```

Comment:
Should we be doing these changes in Kotlin? I think if we switch it should be easy to modify because there is a convert to kotlin tool in Android Studio. Understandable if we want to keep Java for consistency

Reply:
@mrober needs to finish some build config change in order to add Kotlin changes to Crashlytics. And from the conversation with Matt, would be easier to start the conversion from data model first.

Reply:
I purpose let's keep everything in java for now, and once we are ready to work in Kotlin I can re-write some of the class, what do you think?

Reply:
Works for me 👍